src/runtime/pyobject.cs

```diff
@@ -1133,10 +1133,26 @@ public override bool TryBinaryOperation(BinaryOperationBinder binder, Object arg
                     result = null;
                     return false;
             }
-            result = new PyObject(res);
+            result = CheckNone(new PyObject(res));
+
             return true;
         }
 
+        // Workaround for https://bugzilla.xamarin.com/show_bug.cgi?id=41509
+        // See https://github.com/pythonnet/pythonnet/pull/219
+        private static object CheckNone(PyObject pyObj)
+        {
+            if (pyObj != null)
+            {
+                if (pyObj.obj == Runtime.PyNone)
```

Comment:
Doesn't this alter the behaviour? Previously a PyObject wrapping None would be returned, but now that's replaced with null? I must be missing something...